Nodejs/Product/Nodejs/Templates/Files/TypeScriptTsConfig/tsconfig.json

```diff
@@ -0,0 +1,13 @@
+{
+  "compilerOptions": {
+    "noImplicitAny": false,
+    "noEmitOnError": true,
+    "removeComments": false,
+    "sourceMap": true,
+    "target": "es5"
+  },
+  "exclude": [
+    "node_modules",
+    "wwwroot"
```

Comment:
This is an ASP.NET thing. I don't think there's any value in excluding wwwroot for Node.js.